content/issues/2/strangers-in-the-landscape/index.md

```diff
@@ -301,7 +301,12 @@ Since we’re only interested in creating a functional on-screen keyboard, we on
 
 *Fun(?) Note: mixing LTR text with RTL text can lead to extremely confusing UI interactions. For example, in the text input field below, using your mouse, try to highlight the word APPLE plus one character before it and one character after it, i.e. "הAPPLEן". Good luck!*
 
-<input type="text" value="הגדלAPPLEהקטןBANANAסובב" style="font-size: 1.5em; color: #666; width:80%; margin: 0 auto; display: block;">
+
+<input id="tricky-text-selector" type="text" value="הגדלAPPLEהקטןBANANAסובב" style="font-size: 1.5em; color: #666; width:80%; margin: 0 auto; display: block;">
+
+{{< wrap class="sr-only" >}}
+<label for="tricky-text-selector">Tricky text selector</label>
+{{</ wrap >}}
```

Comment:
I made a label and visible only for screen readers. I didn't want to edit anything visible. Does this seem appropriate/ @rlskoeser